loadcaffe.cpp

```diff
@@ -377,6 +469,86 @@ void convertProtoToLuaV2(const caffe::NetParameter &netparam, const char* lua_na
   {
     std::vector<std::pair<std::string, std::string>> lines;
     auto& layer = netparam.layer(i);
+    if(layer.type() == "Deconvolution")
+    {
+      auto &param = layer.convolution_param();
+      bool biasTerm = param.bias_term();
+      int groups = param.group() == 0 ? 1 : param.group();
+      auto &wB = layer.blobs(0);
+      int nInputPlane = (wB.has_shape() ? wB.shape().dim(1) : wB.channels())*groups;
```

Comment:
I don't know whether it is right to exchange `nInputPlane` with `nOutputPlane`. However it actually avoid mistakes in https://github.com/szagoruyko/loadcaffe/blob/master/loadcaffe.lua#L39 . However it also gives wrong loading info in 
https://github.com/szagoruyko/loadcaffe/blob/master/loadcaffe.cpp#L692 . (Specially for FullConvolution, then `printf("%s: %d %d %d %d\n", name, nInputPlane, nOutputPlane, kW, kH)` is right....)